lib/Conversion/ImportVerilog/Structure.cpp

```diff
@@ -191,6 +191,53 @@ struct MemberVisitor {
     return context.convertStatement(procNode.getBody());
   }
 
+  // Handle parameters
+  LogicalResult visit(const slang::ast::ParameterSymbol &paramNode) {
+    auto type = context.convertType(*paramNode.getDeclaredType());
+    if (!type)
+      return failure();
+
+    Value initial;
+    if (const auto *init = paramNode.getInitializer()) {
+      initial = context.convertExpression(*init);
+      if (!initial)
+        return failure();
+    }
```

Comment:
If there is no initializer, I don't think you can create a `NamedConstantOp`, because there is no value to pass to that constant. I think you can just skip/ignore parameters that have no assigned value. So something like:
```suggestion
    const auto *init = paramNode.getInitializer();
    if (!init)
      return success();  // skip parameters without value
    Value initial = context.convertExpression(*init);
    if (!initial)
      return failure();
```